static/index.html

```diff
@@ -131,21 +132,240 @@ <h4>{{ printer_serial }}</h4>
+                                <div class="col-12">
+                                    <div class="row">
+                                        <div class="col-6 px-3">
+                                            <button class="w-100 btn btn-primary" id="light-on">💡 Light on</button>
+                                        </div>
+                                        <div class="col-6 px-3">
+                                            <button class="w-100 btn btn-secondary" id="light-off">💡 Light off</button>
+                                        </div>
+                                    </div>
+                                    <div class="row">
+                                        <div class="col-6 px-3">
+                                            <button class="w-100 btn btn-primary" id="quality-low">
+                                                💾 Quality: low
+                                            </button>
+                                        </div>
+                                        <div class="col-6 px-3">
+                                            <button class="w-100 btn btn-secondary" id="quality-high">
+                                                📀 Quality: high
+                                            </button>
+                                        </div>
+                                    </div>
                                 </div>
                             </div>
-                            <div class="row">
-                                <div class="col-6 px-3">
-                                    <button class="w-100 btn btn-primary" id="quality-low">💾 Quality: low</button>
-                                </div>
-                                <div class="col-6 px-3">
-                                    <button class="w-100 btn btn-secondary" id="quality-high">📀 Quality: high</button>
+                            <!-- Status Container -->
+                            <div class="row mt-3 print-status">
+                                <div class="col-12">
+                                    <div class="row">
+                                        <div class="col-lg-12 px-3 rounded-3">
+                                            <div class="row mt-3">
+                                                <h4>Temperature</h4>
+                                            </div>
+                                            <div class="row mt-2">
+                                                <div class="col-6">
+                                                    <div class="input-group">
+                                                        <div class="input-group-prepend print-temp-label">
+                                                            <div class="input-group-text mx-auto" id="nozzleGroup">
+                                                                &nbsp;<i class="icon-hotend icon"></i>
```

Comment:
Since `.icon` is our own css class, can't we add some spacing attribute to it, instead of this `&nbsp;` 90's layout hack? :smile:

Reply:
Yes basically these are the input groups I was writing about: 
https://getbootstrap.com/docs/5.3/forms/input-group/

Reply:
The hack was actually to force the height of the bootstrap label, spacing wasn't a concern. Every time I took it out, no matter which bootstrap height class I used, the label was smaller than the height of the icon.

Reply:
you'd be surprised by the amount of `@nbsp;` in code to get it to move over a space.

Reply:
&sbsp is now gone with the latest fixes.

Reply:
@just-trey awesome, thanks :)